packages/notifier/src/publish-kit.js

```diff
@@ -382,6 +348,11 @@ const advanceDurablePublishKit = (context, done, value, rejection) => {
  * @param {string} kindName
  */
 export const prepareDurablePublishKit = (baggage, kindName) => {
+  // TODO: Investigate whether memoization is compatible with durability.
```

Comment:
How goes the investigation? Something we should discuss?

Reply:
I found the durable weakmap, but that's a change for a future PR once we have Zones.